lib/common.bash

```diff
@@ -235,3 +235,28 @@ get_pod_config_dir() {
 	pod_config_dir="${BATS_TEST_DIRNAME}/runtimeclass_workloads"
 	info "k8s configured to use runtimeclass"
 }
+
+# Takes two versions in the format of xx.xx.xx and compares them.
+# Versions must contain an equal number of fields.
+#
+# Returns string
+#  0 - first version is greater or versions are the same
+#  1 - second version is greater
+compare_versions() {
+	local -r version_1="$1"
+	local -r version_2="$2"
+
+	fields_1=($(echo $version_1 | tr "." "\n"))
```

Comment:
Nit: You could also make these two variables `local`.

Reply:
Thanks, I changed this